### PR TITLE
chore: add makefile

### PR DESCRIPTION
**Pull request by eddycharly**
Add a makefile.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+############
+# POLICIES #
+############
+
+.PHONY: render-policies
+render-policies: ## Render policies
+	@cd render && go run . -- https://github.com/kyverno/policies/main ../content/en/policies/
```
Comment (chipzoller) [inline Makefile:8]: Needs to have the `-c` flag. Also, need to solve #929 with the `render` utility because it doesn't remove directories.

Comment (eddycharly) [reply Makefile:8]: What is the `-c` flag ?

Comment (eddycharly) [reply Makefile:9]: I added an `rm` statement to remove old content.

Comment (chipzoller) [reply Makefile:9]: `-c` flag means "clean". The `rm` command also works.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**